examples/kubernetes-ingress/scripts/00-create-certs.sh

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/env bash
```

Comment:
Add comments specifying which variables are populated outside of this script and their meaning. See the Bash Style Guide for function comments: https://google.github.io/styleguide/shell.xml#Function_Comments

Reply:
Done